Changelog.md

```diff
@@ -1831,7 +1831,7 @@ Bugfixes:
  * SMTChecker: Fix crash related to statically-sized arrays.
  * TypeChecker: Fix internal error and disallow index access on contracts and libraries.
  * Yul: Properly detect name clashes with functions before their declaration.
- * Yul: Take built-in functions into account in the compilability checker.
+ * Yul: Take built-in functions into account in the compatibility checker.
```

Comment:
This is also incorrect. The `compilability checker` is a component of the compiler that verifies whether all variables are reachable on the stack. See: https://github.com/ethereum/solidity/pull/6063